### ISSUE TITLE
[12.0] Inconsistent text size in PDF reports 

### ISSUE THREAD

**Issue by thomas15v** [odoo/odoo#34525]
Impacted versions:
12

Steps to reproduce:
1. Go to demo.odoo.com
2. Make a new invoice and print this one
3. Now edit the invoice and set "Reference/Description" to a very long tekst. 
4. Print again and compare.

Current behavior:

Both PDF's will have different text size to accommodate the wider tekst. 

Expected behavior:
The length off the tekst should not define the size of the tekst on the PDF. It can over-scan or jump to another line.

Video/Screenshot link (optional):
![image](https://user-images.githubusercontent.com/2163941/60517120-65bc7380-9cdf-11e9-9355-229d6391caf5.png)

**Comment by pedrobaeza** [comment]
Hi, thanks for reporting. That sounds a bit weird...

Are you able to reproduce the problem in runbot infrastructure (http://runbot.odoo.com/runbot - admin/admin)?

**Comment by thomas15v** [comment]
Yeah I can somewhat reproduce it on runbot. You do have to confirm the invoice to make it a noticeable (not sure why).

**Comment by pedrobaeza** [comment]
Link to Odoo runbot record?

**Comment by gdgellatly** [comment]
The problem is probably due to using a Char field like a text field. While there is no realistic upper limit to its size anymore, they are generally expected to be relatively short. I'd say like a lot of things in reports, e.g. big logos, long company names etc generally you end up customising the reports to handle. Probably try widget="text" on the problem field, so the field you want to use like a text field, which isn't a text field, is treated like a text field. See if the works.

**Comment by thomas15v** [comment]
I tried it again on runbot, and this instance seems to replicate exactly what I am experiencing in my development environment:

- [Zoomed invoice](http://554430-12-0-6020f4.runbot27.odoo.com/web?#id=200&action=508&model=account.invoice&view_type=form&menu_id=327) [[**PDF**](https://github.com/odoo/odoo/files/3358094/Draft.Invoice.8.pdf)]
- [Normal Invoice](http://554430-12-0-6020f4.runbot27.odoo.com/web?#id=201&action=508&model=account.invoice&view_type=form&menu_id=327) [[**PDF**](https://github.com/odoo/odoo/files/3358088/Draft.Invoice.10.pdf)]

**Comment by pedrobaeza** [comment]
OK, seen and confirmed that it's happening.

@mart-e not sure how should handle this.

**Comment by C3POdoo** [comment]
Dear @thomas15v,

Thank you for your report but we are closing it due to inactivity.
We apology if we could not look at your request in time.
If your report still makes sense, don't hesitate to reopen a new one. We will try to check it as soon as possible.

_This is an automated message._
